src/test/java/teammates/test/cases/ui/InstructorFeedbackRemindActionTest.java

```diff
@@ -3,21 +3,70 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
+import teammates.common.datatransfer.DataBundle;
+import teammates.common.datatransfer.FeedbackSessionAttributes;
+import teammates.common.datatransfer.InstructorAttributes;
 import teammates.common.util.Const;
+import teammates.common.util.Const.TaskQueue;
+import teammates.ui.controller.InstructorFeedbackRemindAction;
+import teammates.ui.controller.RedirectResult;
 
 public class InstructorFeedbackRemindActionTest extends BaseActionTest {
+    private static final DataBundle dataBundle = getTypicalDataBundle();
     
     @BeforeClass
-    public static void classSetUp() {
+    public static void classSetUp() throws Exception {
         printTestClassHeader();
+        removeAndRestoreTypicalDataInDatastore();
         uri = Const.ActionURIs.INSTRUCTOR_FEEDBACK_REMIND;
     }
     
     @Test
     public void testExecuteAndPostProcess() {
-        // This test case was omitted as the action is executed in the background task queue
-        // so we cannot determine its result.
-          
-        // The logic under the action is covered with test case to ensure the action works.
+        InstructorAttributes instructor1ofCourse1 = dataBundle.instructors.get("instructor1OfCourse1");
+        FeedbackSessionAttributes fs = dataBundle.feedbackSessions.get("session1InCourse1");
+        InstructorFeedbackRemindAction action;
+        RedirectResult rr;
+        
+        ______TS("Unsuccessful case: Not owned Course/Modify Permission, authentication failure");
+        String[] paramsNotOwnedCourse = new String[] {
+                Const.ParamsNames.COURSE_ID, fs.getCourseId(),
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, fs.getSessionName(),
+        };
+        verifyUnaccessibleForInstructorsOfOtherCourses(paramsNotOwnedCourse);
+        verifyUnaccessibleWithoutModifySessionPrivilege(paramsNotOwnedCourse);
```

Comment:
This has been tested in `AllActionsAccessControlTest`, so it needs not be repeated anymore.